docs/user_stories/capacity_calculation_rsc.md

```diff
@@ -0,0 +1,119 @@
+- A loadflow calculation is launched before and after contingency to identify overloads.
+- A sensitivity calculation : for each border, a sensitivity analysis determines the impact on the flow of a small variation of the PST angle, and that for all PSTs, and determines the impact on the flow of a small variation of the generation, and that for all generators.
+
+A cost function is build from the previous results of the sensitivity computation. It is then sent to a solver to find the remedial actions avoiding constraints at a minimal cost.
+
+A secruity analysis is performend at the end of the process to validate the set of remedial actions, found by the optimisation.
+
+All PowSybl features used in this workflow are described below with some implementation examples.
+
+![Workflow](./images/Workflow_Capacity_Calculation_RSC.svg){: width="100%" .center-image}
+
+# Identification of the power system blocks
+
+This user story involves several features from P framework and some other features that are specific:
+
+<img src="./images/File.svg" alt="" style="vertical-align: bottom"/>
+The studied network comes from a set of TSOs' networks. The TSOs' networks can be provided in a common TSO format exchange such as UCTE or CIM-CGMES formats. The following lines of code come from [powsybl-tutorials](https://github.com/powsybl/powsybl-tutorials/tree/master/cgmes) and illustrated this functionality.
+
+```java
+File fileBe = new File(<path_to_file_"MicroGridTestConfiguration_T4_BE_BB_Complete_v2.zip">);
+File fileNl = new File(<path_to_file_"MicroGridTestConfiguration_T4_NL_BB_Complete_v2.zip">);
+```
+
+<br />
+
+<img src="./images/Import.svg" style="vertical-align: bottom"/>
+Each input file is imported with a gateway that transforms the input file in an in-memory object representing the network. The studied network results from the merge of all TSOs' networks.
+
+```java
+Network networkBe = Importers.loadNetwork(fileBe.toString());
+Network networkNl = Importers.loadNetwork(fileNl.toString());
+networkBe.merge(networkNl);
+```
+
+<br />
+
+<img src="./images/Network_merging.svg" style="vertical-align: bottom"/>
+A topological merge of the TSOs' networks is done. The following lines of code come from [powsybl-tutorials](https://github.com/powsybl/powsybl-tutorials/tree/master/cgmes) and illustrated this functionality.
+
+```java
+networkBe.merge(networkNl);
+```
+
+<br />
+
+<img src="./images/Compute_LF.svg" style="vertical-align: bottom"/>
+Then, flows are computed with a load flow simulator. In this tutorial, we use Hades2, which is a non open source software, but available in a freeware mode for experimental purposes. For more details, please visit this [page](https://rte-france.github.io/hades2/features/loadflow.html) to learn about Hades2. A `LoadFlow` object is created through a factory, here a `Hades2Factory`, which needs as input arguments a on-memory network and a computation manager `computationManager` (here defined locally by default).
```

Comment:
Instead of:

> In this tutorial, we use Hades2

I suggest:

> Since there is no fully functional open source load flow simulator integrated to powsybl for the time being, we use Hades2 for the purpose of the tutorial